corehq/apps/data_interfaces/static/data_interfaces/js/case_management.js

```diff
@@ -174,11 +175,11 @@ hqDefine("data_interfaces/js/case_management",[
                     success: function (response) {
                         updateCaseRowCopy(self.selectedCases())();
                         var message = gettext("Successfully copied " + response.copied_cases + " case(s).");
-                        alert_user.alert_user(message, "success");
+                        alert_user.alert_user(message, "success");  // eslint-disable-line
```

Comment:
can you indicate what the issue was with these lines that it can't be resolved except by ignoring the line

Reply:
Right! It's with `alert_user` being camel case. But now that I type this out, I suppose I can just name the import correctly.

Reply:
[ec1898d](https://github.com/dimagi/commcare-hq/pull/33362/commits/ec1898d305246127a8e2318f9dbed1261b9bb23a)